iac-extensions/terraform/src/main/java/org/sonar/iac/terraform/checks/DisabledRDSEncryptionCheck.java

```diff
@@ -30,12 +32,18 @@ public class DisabledRDSEncryptionCheck extends AbstractNewResourceCheck {
   private static final String MESSAGE = "Make sure that using unencrypted databases is safe here.";
   private static final String OMITTING_MESSAGE = "Omitting \"storage_encrypted\" disables databases encryption. Make sure it is safe here.";
   private static final String SECONDARY_MESSAGE = "Related RDS DBInstance";
+  private static final List<String> EXCLUDE_AURORA_ATTRIBUTE = List.of("aurora", "aurora-mysql", "aurora-postgresql");
 
   @Override
   protected void registerResourceConsumer() {
     register("aws_db_instance",
-      resource -> resource.attribute("storage_encrypted")
-        .reportIf(isFalse(), MESSAGE, resource.toSecondary(SECONDARY_MESSAGE))
-        .reportIfAbsent(OMITTING_MESSAGE));
+      resource -> {
+        String engineProperty = Optional.ofNullable(resource.attribute("engine").asString()).orElse("");
+        if (!EXCLUDE_AURORA_ATTRIBUTE.contains(engineProperty)) {
+          resource.attribute("storage_encrypted")
+            .reportIf(isFalse(), MESSAGE, resource.toSecondary(SECONDARY_MESSAGE))
+            .reportIfAbsent(OMITTING_MESSAGE);
+        }
```

Comment:
```suggestion
         AttributeSymbol engine = resource.attribute("engine");
        if (EXCLUDE_AURORA_ATTRIBUTE.stream().anyMatch(auroraAttribute -> engine.is(equalTo(auroraAttribute)))) {
          return;
        }
        resource.attribute("storage_encrypted")
          .reportIf(isFalse(), MESSAGE, resource.toSecondary(SECONDARY_MESSAGE))
          .reportIfAbsent(OMITTING_MESSAGE);
```
As a suggestion to avoid an empty string as a fallback.

Reply:
Yes, sorry I re-did the same mistake as in the other PR.

Reply:
No worry. It's another API.